src/services/anchorService.js

```diff
@@ -1,19 +0,0 @@
-/**
```

Comment:
@jpsantosbh Is this class need? It has the same interface as the anchor service implementation

Reply:
This is a wrapper class that hosts the impl. If every client code is getting the IMPL from the container and not via require(). We don't need it anymore.